django/contrib/admin/utils.py

```diff
@@ -222,6 +222,11 @@ def nested(self, format_callback=None):
         roots = []
         for root in self.edges.get(None, ()):
             roots.extend(self._nested(root, seen, format_callback))
+        # All values are not stored by key None in dict self.edges
+        for key in self.edges:
+            if str(key) != "None":
```

Comment:
If the `edges.get(None, ())` call above works comparing `key` to `None` here should also be working. I also don't know any other Python object which `str()` representation is `'None'` that are not `None` itself.